VirtualTables.sql

```diff
@@ -0,0 +1,161 @@
+CREATE VIEW Tms_TempFrame AS
+(
+SELECT firstTemp.val  AS TempSensorOne,
+       secondTemp.val AS TempSensorTwo,
+       thirdTemp.val  AS TempSensorThree,
+       fourthTemp.val AS TempSensorFour,
+       firstTemp.contextId,
+       firstTemp.receiveTime
+FROM TmsSensorTemp AS firstTemp
+         FULL JOIN (SELECT val, receiveTime, contextId FROM TmsSensorTemp WHERE sensorId = 2) AS secondTemp
+                   ON firstTemp.receivetime = secondTemp.receivetime AND firstTemp.contextid = secondTemp.contextId
+         FULL JOIN (SELECT val, receiveTime, contextId FROM TmsSensorTemp WHERE sensorId = 3) AS thirdTemp
+                   ON firstTemp.receivetime = thirdTemp.receivetime AND firstTemp.contextid = thirdTemp.contextId
+         FULL JOIN (SELECT val, receiveTime, contextId FROM TmsSensorTemp WHERE sensorId = 4) AS fourthTemp
+                   ON firstTemp.receivetime = fourthTemp.receivetime AND firstTemp.contextid = fourthTemp.contextId
+WHERE sensorId = 1
+    );
+
+CREATE VIEW Tms_Fan_Frame AS
```

Comment:
The word "frame" makes me think of a single CAN frame, which could be a bit confusing when people are first trying to learn CAN. Is there another word we could use here?

Reply:
I have been using "Frame" because it is part of a timeframe. I would have used instance but that would simply be wrong. Willing to take other suggestions